src/kanata/windows/interception.rs

```diff
@@ -65,10 +66,28 @@ impl Kanata {
                             };
                             KeyEvent { code, value }
                         }
-                        ic::Stroke::Mouse { state, rolling, .. } => {
+                        ic::Stroke::Mouse {
+                            state,
+                            rolling,
+                            flags,
+                            ..
+                        } => {
                             if mouse_to_intercept_hwids.is_some()
                                 || mouse_to_intercept_excluded_hwids.is_some()
                             {
+                                if is_device_interceptable(
+                                    dev,
+                                    &intrcptn,
+                                    &mouse_to_intercept_hwids,
+                                    &mouse_to_intercept_excluded_hwids,
+                                    &mut is_dev_interceptable,
+                                ) {
+                                    if let Some(ms_mvmt_key) = *mouse_movement_key.lock() {
```

Comment:
Curious what the thought is behind doing this in both branches? Since having a copy in the `else` means all mouse devices get involved in this check, seems there is no need to filter it here; meaning there could be deduplication by moving this between lines 77,78.

Reply:
if there are include or exclude lists, we need to check the event's origin against them. no check is needed if there are no lists. but yes, the logic can be restructured to avoid duplicating the send.

Reply:
There are conflicting motivations in the code:
- else branch: ignore that the user has not asked for mouse devices to be intercepted, check all mouse events
- within this branch: filter to only devices that the user has specified

I would say let's eliminate the code in the `else` branch down at line 107.